README.md

```diff
@@ -1,40 +1,85 @@
 # Fortnox.NET
+
 .NET bindings for the <a href="https://developer.fortnox.se/documentation/">Fortnox</a> API.
- 
+
 ## Usage
+
 To access the Fortnox API you need an Access Token and Client Secret: <a href="https://developer.fortnox.se/general/authentication/">https://developer.fortnox.se/general/authentication/</a>.
- 
+
 The unit tests in this repository serves as a way for us to make sure our code works as well as examples of how to call the different routes that the Fortnox API provides. There are mainly two ways to make a request using this SDK, one for requesting a list of resources and one for requesting a single resource.
- 
+
 ### Listed resource
+
 When requesting a list of resources you create a ListRequest object of the type of resource you are interested in.
- 
+
 ```CSharp
 var request = new OrderListRequest("ACCESS-TOKEN", "CLIENT-SECRET");
 var orders = await OrderService.GetOrdersAsync(request);
 ```
- 
+
 You can also add filters and ordering to routes that accepts it, for example:
- 
+
 ```CSharp
 var request = new EmployeeListRequest("ACCESS-TOKEN", "CLIENT-SECRET")
 {
     Filter = EmployeeFilters.Active
 };
 var employeeList = await EmployeeService.GetEmployeesAsync(request);
 ```
- 
+
 ### Single resource
+
 When requesting a single resource you create a FortnoxApiRequest object and pass that into the service to make your request.
- 
+
 ```CSharp
 var request = new FortnoxApiRequest("ACCESS-TOKEN", "CLIENT-SECRET");
 var response = await ArticleService.GetArticleAsync(request, "100370");
 ```
- 
+
+### WebSocket
```

Comment:
WebHooks, not WebSockets

Reply:
It's interesting that they are using the WebSockets themselves. It uses the same protocol as weboskcets i believe, but that server-to-server implementation is called WebHooks

Thoughts on this @brokenprogrammer @BaconFinger

Reply:
I wrote webhooks at first but then when I scimmed through the documentation once more the title said WebSocket API. You sure you want to name it WebHooks?

Reply:
I'm very ambivalent about this. Personally I'd say go for WebSockets since that's the tech being used, but either works. 

Reality is it probably doesn't matter what we call it cause the terms are so similar everyone is just going to use them interchangeably.

Reply:
Let's kepp it as is